planning_launch/config/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/occlusion_spot.param.yaml

```diff
@@ -0,0 +1,34 @@
+/**:
+  ros__parameters:
+    occlusion_spot:
+      detection_method: "predicted_object"  # [-] candidate is "occupancy_grid" or "predicted_object"
+      pass_judge: "current_velocity"        # [-] candidate is "current_velocity"
```

Comment:
@k-obitsu 
can you sync this part please :pray: 
![image](https://user-images.githubusercontent.com/65527974/162414608-b2373815-60ec-4a20-92c5-01d4116cf512.png)